assignments/this.js

```diff
@@ -1,26 +1,71 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
+* 1. Window Binding - Window binding uses this in the loosest. It is used to describe the window that it is in. In this case it is the web browser object.
+* 2. Implicit - Implicit binding is an implied or automatic when there is only one possiblity and it is obvious in context it is an implict instance.
+* 3. Explicit - Explicit binding is a more specific instance of :this: because it is not automatic it must be further specified what :this: is refering to.
+* 4. Prototyple - Prototyple programming is defiently the most complex of the principles. When using a prototyple instance of :this: involves constructor functions and passing :this: from one object to another using methods and/or functions.
+
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
 // code example for Window Binding
+console.log(this);
 
 // Principle 2
 
 // code example for Implicit Binding
+const realMikeKerbleski = {
+  'action' : 'stand up',
+  'askTheQuestion': function(name) {
+  console.log(`will the real ${name} please ${this.action}?`);
+  }
+}
+realMikeKerbleski.askTheQuestion("MikeKerbleski");
 
 // Principle 3
 
 // code example for New Binding
 
+const fakeMikeKerbleski = {
```

Comment:
This looks like it should be the example for explicit binding. Did they get switched? This and the next example are both great examples but swapped!